src/OSPSuite.Core/Domain/Exceptions.cs

```diff
@@ -149,4 +149,11 @@ public CannotLockFileException(Exception exception) : base(exception.Message)
       {
       }
    }
+
+   public class ContainerNotFoundException : OSPSuiteException
```

Comment:
hum. I don't understand why we generate a specific  exception that does nothing. It's like a marker exception somehow but it's sued only once and caught in the same file. I don't think we need this and we can throw the original exception by adding the building block name no?

Reply:
Thought about it, but when this new exception is thrown, we do not have all the data we need, I. E SpatialStructure.
So the execption will not have this information that is critical to this task.
Therefore we could either send this parameter in, but it will only be used ONLY for this exception, which will not always be thrown.
The other thing we could do is throw it and retrhrow the same exception overwritting the message.
SO let me know your thoughts about this.